OutOfSchool/Tests/OutOfSchool.WebApi.IntegrationTests/OutOfSchool.WebApi.IntegrationTests.csproj

```diff
@@ -27,6 +27,7 @@
     </ItemGroup>
 
     <ItemGroup>
+        <ProjectReference Include="..\..\OutOfSchool.AuthorizationServer\OutOfSchool.AuthorizationServer.csproj" />
```

Comment:
I don't see any new tests/code that would require this reference, what is it for?

Reply:
Yes, that's wrong dependency. Removed